### PR TITLE
Add trust-remote-code flag to handle remote tokenizers

### PR DESCRIPTION
**Pull request by codethazine**
Fix issue #353

### CODE REVIEW

File: vllm/transformers_utils/tokenizer.py [hunk 1]
```diff
@@ -31,15 +32,29 @@ def get_tokenizer(
...
         # The LLaMA tokenizer causes a protobuf error in some environments.
         err_msg = (
             "Failed to load the tokenizer. If you are using a LLaMA-based "
             f"model, use '{_FAST_LLAMA_TOKENIZER}' instead of the original "
             "tokenizer.")
         raise RuntimeError(err_msg) from e
+    except ValueError as e:
+        # If the error pertains to the tokenizer class not existing or not currently being imported, suggest using the --trust-remote-code flag.
+        if e is not None and "does not exist or is not currently imported." in str(
+                e):
```
Comment (WoosukKwon) [inline vllm/transformers_utils/tokenizer.py:50]: Please fix this part according to our formatter.

File: vllm/transformers_utils/tokenizer.py [hunk 2]
```diff
@@ -31,15 +32,29 @@ def get_tokenizer(
...
+            err_msg = (
+                "Failed to load the tokenizer. If the tokenizer is a custom "
+                "tokenizer, not yet available in the HuggingFace transformers "
```
Comment (WoosukKwon) [inline vllm/transformers_utils/tokenizer.py:53]: nit
```suggestion
                "tokenizer not yet available in the HuggingFace transformers "
```

**Review by WoosukKwon — CHANGES_REQUESTED** [review]
Hi @codethazine, Thanks for your contribution! The PR generally looks good. Left some minor comments.

**Comment by codethazine** [comment]
The formatting is now fixed.

**Review by zhuohan123 — APPROVED** [review]
LGTM! Thank you for your contribution!

**Pull request merged**